apps/web/src/pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,17 @@
+import NextAuth from 'next-auth';
+import TwitterProvider from 'next-auth/providers/twitter';
+
+export default NextAuth({
+  callbacks: {
+    session({ session }) {
+      return session;
+    },
+  },
+  providers: [
+    TwitterProvider({
+      clientId: process.env.TWITTER_CLIENT_ID || '',
```

Comment:
why the fallback?

Reply:
it's just because typescript harassing me without it lol

![image](https://user-images.githubusercontent.com/5343791/159138751-1e20caf4-a9ad-4876-9761-59049ed38bd6.png)